src/api/lib/tasks/dev.rake

```diff
@@ -108,7 +108,7 @@ namespace :dev do
           end
         end
       end
-    end
+    end unless Rails.env.production?
```

Comment:
Thanks for the patch:smile: But please move this to the beginning of the :rubocop definition, like
```ruby
unless Rails.env.production?
  namespace :rubocop do
...
end
```

That way it's clear right from the start when this task is available and when it's not.

Reply:
I won't today

Reply:
Okay. I solved it